test/e2e/app-dir/app/index.test.ts

```diff
@@ -1482,14 +1482,19 @@ describe('app dir - basic', () => {
 
     describe('should support React fetch instrumentation', () => {
       it('server component', async () => {
+        // trigger compilation of 404 here first.
+        // Any other page being compiled between refresh of a page would get us fresh modules i.e. not catch previous regressions where we restored the wrong fetch.
+        await next.browser('/_not-found')
         const browser = await next.browser('/react-fetch/server-component')
         const val1 = await browser.elementByCss('#value-1').text()
         const val2 = await browser.elementByCss('#value-2').text()
+        expect(val1).toBe(val2)
 
-        // TODO: enable when fetch cache is enabled in dev
-        if (!isDev) {
-          expect(val1).toBe(val2)
-        }
```

Comment:
A bit odd that this test was disabled in dev. @timneutkens was this just an oversight back then?

Reply:
Yea this was just leftover and not updated way back it seems